packages/calcite-components/src/components/block/block.tsx

```diff
@@ -334,14 +382,42 @@ export class Block
     ) : null;
   }
 
+  renderIcon(iconPosition: string): VNode {
```

Comment:
Let's type the position argument as it doesn't take any string. Also, it can probably be simplified to `position` since we know it's icon-related due to the method name.